lib/ui/popover/popover.js

```diff
@@ -44,12 +44,26 @@
     this.destroy = function() {
       $element.popover('destroy');
     };
+
+    if($scope.showOnLoad) {
+      this.show();
+      if($scope.delay && $scope.delay.hide) {
+        $timeout(this.hide, $scope.delay.hide);
+        return;
+      }
+      // If no delay is found or cannot be parsed, set a default timeout so that the popover doesn't stick around forever
+      $timeout(this.hide, 10000);
```

Comment:
I would probably move out the default delay `10000` into some kind of configuration that can be modified by other users of the SDK.  

Here is an example that uses a provider for configuring all dropdowns globally across the app [avDropdownConfig ](https://github.com/Availity/availity-angular/blob/31e184a27ed5eadd739c935900c3eec869716d07/lib/ui/dropdown/dropdown.js#L7)